maven-plugin-tools-api/pom.xml

```diff
@@ -125,4 +130,30 @@
       </plugin>
     </plugins>
   </build>
+  <profiles>
+    <profile>
+      <id>run-its</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-failsafe-plugin</artifactId>
+            <executions>
+              <execution>
+                <goals>
+                  <goal>integration-test</goal>
+                  <goal>verify</goal>
+                </goals>
+                <configuration>
+                  <!-- don't run ITs against all (non-LTS) JDK javadocs -->
+                  <excludedGroups>nonLtsJavadocs</excludedGroups>
```

Comment:
If we here exclude tests with this tag - where such tests will be executed?

Reply:
only if you override that parameter or execute that IT in your IDE

Reply:
always testing against all ever released JDKs just takes too much time. It is rather advisable to execute that IT manually in case something was fundamentally changed about link creation which might affect all versions.

Reply:
Any other concerns @slawekjaranowski?

Reply:
ok. I see test are execute - only not the non lst are extracted to other test